### DISCUSSION TITLE
Improve search experience by supporting bookmarks and automatic colorizing of log entries

### DISCUSSION THREAD

**Discussion by Andrej-Peplinski** [datalust/seq-tickets#1857] [Ideas]
I would be great if we could bookmark individual log entries and then jump between them. Ideally, we would be able to optionally name the bookmark so that we can quickly see why this entry has been marked.

Additionally, it would be nice if we could highlight log entries (e.g. by changing the background color) based on some search criterial (e.g. contains a text or a specific property).

**Comment by nblumhardt** [comment]
Thanks for the suggestions @Andrej-Peplinski; these line up with some things we've been considering for a while - definitely a direction we're keen to explore 👍
